services/provider/proto/provider.proto

```diff
@@ -73,6 +73,10 @@ message ExternalResource {
     string kind = 2;
     // Data contains the contents of the external cluster resource
     bytes Data = 3;
+    // Labels contains the labels of the external cluster resource
+    bytes Labels = 4;
+    // Annotations contains the annotations of the external cluster resource
+    bytes Annotations = 5;
```

Comment:
Why bytes? Labels and annotations have a known structure of a string key-value pair list.
Let do proper encoding for it